src/python/WMCore/MicroService/MSOutput/MSOutputTemplate.py

```diff
@@ -102,6 +102,7 @@ def docSchema(self):
                            'Copies': 1,
                            ...}],
                     "TransferStatus": "pending"|"done,
+                    "DBSUpdateStatus": "pending"|"done,
```

Comment:
I guess I would make it a boolean, thus something like:
```
"DBSStatusUpdated": False|True,
```
but then it would require more changes from your side. So I leave it up to you.

Reply:
Good suggestion, this is applied in the last commit.